stdlib/docs/development.md

```diff
@@ -66,13 +66,24 @@ We provide a simple Bash script to build the standard library package and
 `test_utils` package that is used by the test suite.
 
 Just run `./stdlib/scripts/run-tests.sh` which will produce the necessary
-`mojopkg` files inside your `build` directory, and then run `lit -sv
-stdlib/test`.
+`mojopkg` files inside your `build` directory, after this is done, the script will
+run all the tests automatically.
 
 ```bash
 ./stdlib/scripts/run-tests.sh
+```
+
+If you wish to run the unit tests that are in a specific test file, you can do 
```

Comment:
FYI you have trailing spaces in this file that your new markdownlint would catch here :)

Reply:
I think I'll wait until the pre-commit is updated in nightly. I'm too lazy to do the git gymnastic or to install the tool myself

Reply:
No worries, I fixed them up internally and landed this.